challenges/classes.js

```diff
@@ -1,7 +1,28 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+/*CuboidMaker.prototype.volume = function(){
+    return this.length*this.width*this.height;
+}*/
 
+  class volume extends CuboidMaker{
+      constructor(properties){
+      super(properties)}
+      cuboid(){
+        return this.length*this.width*this.height;
+      }
+  }
+
+/*CuboidMaker.prototype.surfaceArea = function(){
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+  }*/
+  class surfaceArea extends CuboidMaker{
+    constructor(properties){
+    super(properties)}
+    cuboid(){
+        return 2 * (this.length * this.width + this.length * this.height + this.width * this.height)
+    }
+}
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+ console.log(cuboid.volume()); // 100
+ console.log(cuboid.surfaceArea()); // 130
 
```

Comment:
This is getting errors that CuboidMaker is not defined. Where are class surface area and volume extending from?